docs/source/whatsnew/pr/end-shortcut-accept-suggestion.rst

```diff
@@ -0,0 +1,5 @@
+Added shortcut for accepting auto suggestion
+============================================
+
+Added End key shortcut for accepting auto-suggestion
```

Comment:
Would be nice to mention that the binding works in Vi mode too, provided `TerminalInteractiveShell.emacs_bindings_in_vi_insert_mode` is set.

Reply:
you are right. I adapted accordingly